helm/eirini/templates/bits.yaml

```diff
@@ -110,9 +110,13 @@ spec:
         - name: bits-cert
           secret:
             secretName: private-registry-cert
+        - name: bits-assets
+          hostPath:
+            path: /bits/assets/
+            type: DirectoryOrCreate
       containers:
       - name: bits
-        image: flintstonecf/bits-service:2.25.0-dev.7
+        image: flintstonecf/bits-service:latest
```

Comment:
I think we want to avoid having any references to :latest because we don’t want users to silently get a different version than they thought they were installing when they applied the helm chart.